common/app/model/Cached.scala

```diff
@@ -80,19 +82,32 @@ object Cached extends implicits.Dates {
       cacheableResult.result
     }
 
-  private def cacheHeaders(seconds: Int, result: Result, maybeHash: Option[(Hash, Option[String])]) = {
+  /*
+    NOTE, if you change these headers make sure they are compatible with our Edge Cache
+
+    see
+    http://tools.ietf.org/html/rfc5861
+    http://www.fastly.com/blog/stale-while-revalidate
+    http://docs.fastly.com/guides/22966608/40347813
+
+    This explains Surrogate-Control vs Cache-Control
+    TLDR Surrogate-Control is used by the CDN, Cache-Control by the browser - do *not* add `private` to Cache-Control
+    https://docs.fastly.com/guides/tutorials/cache-control-tutorial
+  */
+  private def cacheHeaders(maxAge: Int, result: Result, maybeHash: Option[(Hash, Option[String])]) = {
     val now = DateTime.now
-    val expiresTime = now + seconds.seconds
-    val maxAge = if (DoubleCacheTimesSwitch.isSwitchedOn) seconds * 2 else seconds
+    val expiresTime = if (LongCacheSwitch.isSwitchedOn) now + min(maxAge, 60).seconds else now + maxAge.seconds
 
-    // NOTE, if you change these headers make sure they are compatible with our Edge Cache
+    val staleWhileRevalidateSeconds = max(maxAge / 10, 1)
+    val surrogateCacheControl = s"max-age=$maxAge, stale-while-revalidate=$staleWhileRevalidateSeconds, stale-if-error=$tenDaysInSeconds"
 
-    // see
-    // http://tools.ietf.org/html/rfc5861
-    // http://www.fastly.com/blog/stale-while-revalidate
-    // http://docs.fastly.com/guides/22966608/40347813
-    val staleWhileRevalidateSeconds = math.max(maxAge / 10, 1)
-    val cacheControl = s"max-age=$maxAge, stale-while-revalidate=$staleWhileRevalidateSeconds, stale-if-error=$tenDaysInSeconds"
+    val cacheControl = if (LongCacheSwitch.isSwitchedOn) {
+      val browserMaxAge = min(maxAge, 60)
```

Comment:
Why make the browser's max age smaller than the surrogate max age? Is it because this approach supports purging whilst giving longer CDN-level cache times? Also, when the system switches from long-cache = off to long-cache = on, browsers could hold on to the bigger max-age for a bit.